packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -340,4 +361,135 @@ class DeferredNode extends AttributedNode {
         resolver(refId).applyAttributes(attributes);
     concreteRef.build(builder, transform);
   }
+
+  @override
+  void visitChildren(NodeCallback visitor) {}
+}
+
+/// A leaf node in the tree that represents inline text.
+///
+/// Leaf nodes get added with all paint and transform accumulations from their
+/// parents applied.
+class TextNode extends AttributedNode {
+  /// Create a new [TextNode] with the given [text].
+  TextNode(
+    this.text,
+    this.baseline,
+    this.absolute,
+    SvgAttributes attributes,
+  ) : super(attributes);
+
+  /// The text this node contains.
+  final String text;
+
+  /// The x, y coordinate of the starting point of the text baseline.
+  final Point baseline;
+
+  /// Whether the [baseline] is in absolute or relative units.
+  final bool absolute;
+
+  Paint? _paint(Rect bounds, AffineMatrix transform) {
+    final Fill? fill = attributes.fill?.toFill(bounds, transform);
+    final Stroke? stroke = attributes.stroke?.toStroke(bounds, transform);
+    if (fill == null && stroke == null) {
+      return null;
+    }
+    return Paint(
+      blendMode: attributes.blendMode,
+      fill: fill,
+      stroke: stroke,
+    );
+  }
+
+  static final Map<String, double> _kTextSizeMap = <String, double>{
```

Comment:
please add a comment explaining where these values come from (I think the spec?)

Reply:
how about I delete it and use the existing map :)